app/controllers/users/mfa_selection_controller.rb

```diff
@@ -24,6 +24,9 @@ def update
         @presenter = two_factor_options_presenter
         render :index
       end
+    rescue ActionController::ParameterMissing
```

Comment:
Do you have a link to the error? This isn't one we typically have to rescue

Reply:
https://one.newrelic.com/nr1-core/errors/overview/MTM3NjM3MHxBUE18QVBQTElDQVRJT058NTIxMzY4NTg?account=1376370&duration=43200000&state=cf70abbd-9c61-e89a-4214-de542b0ad3f8